### PR TITLE
[PATCH v3] example: improve example application documentations

### PR DESCRIPTION
**Pull request by MatiasElo**

### CODE REVIEW

File: example/time/time_global_test.c [hunk 1]
```diff
@@ -4,6 +4,14 @@
  * SPDX-License-Identifier:     BSD-3-Clause
  */
 
+/**
+ * @example time_global_test.c
```
Comment (TuomasTaipale) [inline example/time/time_global_test.c:8]: Is this visible if you generate it locally?

File: test/performance/odp_l2fwd.c [hunk 2]
```diff
@@ -6,6 +6,14 @@
  * SPDX-License-Identifier:     BSD-3-Clause
```
Comment (TuomasTaipale) [inline test/performance/odp_l2fwd.c:6]: Unrelated, but should we ditch the "example" l2fwd which is just a symbolic link and just have the "performance" l2fwd?

File: example/time/time_global_test.c (hunk 1 above)
Comment (MatiasElo) [reply example/time/time_global_test.c:8]: Good catch, I was wondering the same thing at some point but completely forgot it. Perhaps the cleanest fix is to rename the application to e.g. odp_time_global_test, so it matches Doxyfile FILE_PATTERNS.

File: test/performance/odp_l2fwd.c (hunk 2 above)
Comment (MatiasElo) [reply test/performance/odp_l2fwd.c:6]: Agreed, I updated the card regarding example application clean up.

File: example/time/time_global_test.c (hunk 1 above)
Comment (MatiasElo) [reply example/time/time_global_test.c]: Renamed to `odp_time_global_test.c ` in V2.

**Review by TuomasTaipale — APPROVED** [review]

**Pull request merged**